massa-storage/src/lib.rs

```diff
@@ -139,7 +189,17 @@ impl Storage {
         });
         let mut operations = self.operations.write();
         for id in operation_ids {
-            operations.remove(id);
+            let (prefix, suffix) = id.split();
+            let len = match operations.get_mut(&prefix) {
```

Comment:
maybe use `entry` so as to remove it in the same operation if necessary?

Reply:
done